osu.Game/Screens/Play/BreakOverlay.cs

```diff
@@ -109,6 +123,57 @@ protected override void LoadComplete()
             initializeBreaks();
         }
 
+        protected override void Update()
+        {
+            base.Update();
+
+            updateBreakTimeBindable();
+        }
+
+        private void resetBreakIndex()
+        {
+            isBreakTime.Value = false;
+            currentBreakIndex = 0;
+        }
+
+        private void updateBreakTimeBindable()
+        {
+            if (breaks?.Any() != true)
+            {
+                resetBreakIndex();
```

Comment:
Is this required, considering the input of `breaks` is already governed by the property setter?